src/content_scripts/autoconsent.js

```diff
@@ -11,11 +11,38 @@
 
 import AutoConsent from '@duckduckgo/autoconsent';
 
+const FLAG_STORAGE_KEY = 'ghostery:autoconsent';
```

Comment:
One problem that I see is that it can be used for adblocker detection (by anti-adblockers) and for fingerprinting (trackers).

I'm not aware of a straightforward technique to transfer information in a safe and non-detectable way. So, I cannot offer an constructive advice. Message passing might be the best choice.

Reply:
That's a good point, I didn't take into account. Ill search if there is another way to store information in sync with browsing history of the user.